src/EventTypeDefs.ts

```diff
@@ -53,10 +56,15 @@ export type ControlCursorCallback = ControlCallback<string>;
  * relative to target's containing coordinate plane
  * both agree on every point
  */
-export type Transform = {
-  target: FabricObject;
+export type Transform<T extends FabricObject = FabricObject> = {
+  target: T;
   action?: string;
-  actionHandler?: TransformActionHandler;
+  actionHandler?: (
+    eventData: TPointerEvent,
+    transform: Transform<T>,
```

Comment:
any clues on handling this cyclic declaration @jiayihu ?

Reply:
What is the issue with leaving `TransformActionHandler` here?